.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,26 @@
+
+## 👀 4. 리뷰 중점 사항
+
+<!-- 논의할 사항이나 중점적으로 리뷰 받고 싶은 사항을 작성해주세요 -->
+
+- [ ]
+
+## 🧪 5. 테스트
+
+- [ ]
+
+**테스트 방법**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)